js/packages/teams-ai/src/StreamingResponse.ts

```diff
@@ -140,6 +143,7 @@ export class StreamingResponse {
                 return {
                     type: 'message',
                     text: this._message,
+                    attachments: this._attachments,
```

Comment:
Does it matter that this._attachments can be null?  Or am I reading this wrong?

Reply:
nope,  type for attachments can be `Attachment[] | undefined`